demonstrations/tutorial_test_1.py

```diff
@@ -0,0 +1,272 @@
+        variational quantum eigensolver algorithm.
+    :property="og:image": https://pennylane.ai/qml/_images/pes_h2.png
+
+.. related::
+
+   tutorial_quantum_chemistry Building molecular Hamiltonians
+   tutorial_vqe_parallel VQE with parallel QPUs
+   tutorial_vqe_qng Accelerating VQE with the QNG
+   tutorial_vqe_spin_sectors VQE in different spin sectors
+   tutorial_vqt Variational quantum thermalizer
+
+*Author: PennyLane dev team. Last updated: 15 July 2021.*
+
+The Variational Quantum Eigensolver (VQE) is a flagship algorithm for quantum chemistry
+using near-term quantum computers [#peruzzo2014]_. It is an application of the
+`Ritz variational principle <https://en.wikipedia.org/wiki/Ritz_method>`_, where a quantum
+computer is trained to prepare the ground state of a given molecule.
+
+The inputs to the VQE algorithm are a molecular Hamiltonian and a
+parametrized circuit preparing the quantum state of the molecule. Within VQE, the
+cost function is defined as the expectation value of the Hamiltonian computed in the
+trial state. The ground state of the target Hamiltonian is obtained by performing an
+iterative minimization of the cost function. The optimization is carried out
+by a classical optimizer which leverages a quantum computer to evaluate the cost function
+and calculate its gradient at each optimization step.
+
+In this tutorial you will learn how to implement the VQE algorithm in a few lines of code.
+As an illustrative example, we use it to find the ground state of the hydrogen
+molecule, :math:`\mathrm{H}_2`. First, we build the molecular Hamiltonian using a minimal
+basis set approximation. Next, we design the quantum circuit preparing the trial
+state of the molecule, and the cost function to evaluate the expectation value
+of the Hamiltonian. Finally, we select a classical optimizer, initialize the
+circuit parameters, and run the VQE algorithm using a PennyLane simulator.
+
+Let's get started!
+
+Building the electronic Hamiltonian
+-----------------------------------
+The first step is to specify the molecule we want to simulate. This
+is done by providing a list with the symbols of the constituent atoms
+and a one-dimensional array with the corresponding nuclear coordinates
+in `atomic units <https://en.wikipedia.org/wiki/Hartree_atomic_units>`_.
+"""
+import numpy as np
+
+symbols = ["H", "H"]
+coordinates = np.array([0.0, 0.0, -0.6614, 0.0, 0.0, 0.6614])
+
+##############################################################################
+# The molecular structure can also be imported from an external file using
+# the :func:`~.pennylane_qchem.qchem.read_structure` function.
+#
+# Now, we can build the electronic Hamiltonian of the hydrogen molecule
+# using the :func:`~.pennylane_qchem.qchem.molecular_hamiltonian` function.
+
+import pennylane as qml
+
+H, qubits = qml.qchem.molecular_hamiltonian(symbols, coordinates)
+print("Number of qubits = ", qubits)
```

Comment:
```suggestion
print("Number of qubits = ", qubits) 
```
No new line in the code either